### ISSUE TITLE
Include in some way the level of expertise needed

### ISSUE THREAD

**Issue by Andresfgomez970** [desireevl/awesome-quantum-computing#73]
Hello everyone! First of all, thanks for all the fantastic resources. 

I think it would be handy to include the level of expertise needed in some way. That would help to motivate to carry out an organized reading plan. However, not being an expert, I would love to hear some suggestions and thoughts. 

I have already read https://quantum.country/qcvc and took quantum mechanics at the University. I would love to contribute to the present issue (or leave it aside) based on your suggestions.

**Comment by desireevl** [comment]
Hi, thanks for the idea! Are you thinking of creating a sort of tag for each resource based on the level of expertise needed? Like beginner, intermediate, expert etc and then creating a reading plan based on these?

I haven't read fully through each resource on the list, so if this is what you are thinking, then it would need to be a combined effort to go through each resource to categorise them and create a reading list. 

Let me know what you are thinking!

**Comment by Andresfgomez970** [comment]
Yes, overall, I am worried about a good reading plan. I think this is key to advancing in any field and would make all these resources more valuable. 

I like the proposed tags. We could give them the following meaning:  

Beginner
High school background. E.g., references to cope with complex numbers, calculus, probabilistic theory, linear algebra, and fundamental physics concepts.

Elementary
The above conditions are checked. Some good books for a general perspective of the field. Let us say "Quantum Mechanics since Democritus." Along with them, resources for a solid base in quantum mechanics and basic programming. For instance, reading https://quantum.country/ along https://qiskit.org/textbook/preface.html. Then more advanced books like  A Short Introduction to Quantum Information and Quantum Computation by Michel Bellac.

Intermediate
Books that dive deeper into specific fields. Let's say quantum algorithm development, the realization of quantum computers, quantum information theory, complexity theory with quantum computers, etc. I also imagine here more complex applications in some given frameworks.

Expert
I imagine a very defined line of research, with some suggested authors, papers, groups, projects, etc.

Again, I am not an expert in the field. So, I would appreciate any thoughts. I am about to begin reading in the intermediate phase.

**Comment by desireevl** [comment]
Yes that sounds like a good idea. Please feel free to fork this repository and make the changes :) We can discuss merging your ideas into this main repo when you are ready!
